### PR TITLE
Initial provider review - changes

### PR DESCRIPTION
**Pull request by ewilde**
This PR references the issues raised in #27

### CODE REVIEW

File: scripts/goimportscheck.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Check goimports
+echo "==> Checking that code complies with goimports requirements..."
+goimports_files=$(goimports -l `find . -name '*.go' | grep -v vendor`)
+if [[ -n ${goimports_files} ]]; then
+    echo 'goimports needs running on the following files:'
+    echo "${goimports_files}"
+    echo "You can use the command: \`make fmt\` to reformat code."
```
Comment (kevholditch) [inline scripts/goimportscheck.sh]: message is wrong here should be `make goimports`

File: runscope/data_source_runscope_integration_test.go [hunk 2]
```diff
@@ -4,21 +4,22 @@ import (
 	"fmt"
 	"testing"
```
Comment (paultyng) [inline runscope/data_source_runscope_integration_test.go]: not a big deal here, `goimports` sometimes doesn't quite get the whitespacing right as it errs on the side of including your import groups, but I'd group `os` with the other stdlib packages normally, but won't hold up process on this.

File: runscope/data_source_runscope_integration_test.go [hunk 3]
```diff
@@ -57,7 +58,7 @@ data "runscope_integration" "by_type" {
 
 func TestAccDataSourceRunscopeIntegration_Filter(t *testing.T) {
 
-	teamId := os.Getenv("RUNSCOPE_TEAM_ID")
+	teamIS := os.Getenv("RUNSCOPE_TEAM_ID")
```
Comment (paultyng) [inline runscope/data_source_runscope_integration_test.go]: this may be a typo?

File: runscope/provider_test.go [hunk 4]
```diff
@@ -4,11 +4,12 @@ import (
 	"os"
 	"testing"
```
Comment (paultyng) [inline runscope/provider_test.go]: Similar issue here with extra whitespace, but not a big deal.

**Comment by ewilde** [comment]
@paultyng thanks for the review, i've updated where you have commented. What are the next steps?

File: website/docs/d/integrations.html.markdown [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+layout: "runscope"
+page_title: "Runscope: runscope_integrations"
+sidebar_current: "docs-runscope-datasource-integrations"
+description: |-
+  Get information about runscope integrations enabled on for your team.
+---
+
+# runscope\_integration
+
+Use this data source to list all of your [integration]s(https://www.runscope.com/docs/api/integrations)
```
Comment (paultyng) [inline website/docs/d/integrations.html.markdown]: Not sure if this `s` needs to be inside the brackets? or the URL next to the brackets?

File: website/docs/d/integrations.html.markdown [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `filter` - (Optional) Filter to reduce the list of integrations returned.
+
+Variables (`filter`) supports the following:
+
```
Comment (paultyng) [inline website/docs/d/integrations.html.markdown:33]: Missing some copy here, if you are not ready to add content, can just remove this paragraph.

**Comment by paultyng** [comment]
Sorry had one more minor catch in the markdown. Will follow up in email about next steps.

**Comment by cgriggs01** [comment]
Hi @ewilde,

PR looks great, when do you plan to merge?

Best,
Chris

**Comment by ewilde** [comment]
@cgriggs01 thanks for the feedback i've updated the integration docs and merged the PR

**Pull request merged**